src/main/java/co/inventorsoft/academy/dao/MatchDAO.java

```diff
@@ -0,0 +1,71 @@
+package co.inventorsoft.academy.dao;
+
+import co.inventorsoft.academy.dao.intetface.MatchDAOInterface;
+import co.inventorsoft.academy.jdbc.MyJDBC;
+import co.inventorsoft.academy.model.Match;
+import org.springframework.stereotype.Repository;
+
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+import static co.inventorsoft.academy.jdbc.MyJDBC.connection;
+
+@Repository
+public class MatchDAO implements MatchDAOInterface {
+    private static final String INSERT_MATCH = "INSERT INTO matches (round, team1, team2, points_team1, points_team2) VALUES (?, ?, ?, ?, ?) RETURNING id;";
+    private static final String SELECT_MATCH_BY_ID = "SELECT round, team1, team2, points_team1, points_team2 FROM matches WHERE id = ?;";
+
+    /**
+     * This method add {@link Match} to database
+     *
+     * @param match input {@link Match}
+     */
+    @Override
+    public int addMatch(Match match) {
+        int result = -1;
+        try (PreparedStatement preparedStatement = connection.prepareStatement(INSERT_MATCH)) {
+            preparedStatement.setString(1, match.getRound());
+            preparedStatement.setInt(2, match.getTeam1());
+            preparedStatement.setInt(3, match.getTeam2());
+            preparedStatement.setInt(4, match.getPointsTeam1());
+            preparedStatement.setInt(5, match.getPointsTeam2());
+            ResultSet rs = preparedStatement.executeQuery();
+            while (rs.next()) {
+                result = rs.getInt("id");
+            }
+            rs.close();
```

Comment:
If possible use try with resources.

Reply:
I can do that because I  set value to preparedStatement and than I  use preparedStatement.executeQuery();

Reply:
```
try (PreparedStatement preparedStatement = connection.prepareStatement(INSERT_MATCH)) {
    // ...

    try (ResultSet rs = preparedStatement.executeQuery()) {
        // ...
    }
}
```